src/client/components/SearchFilterComponent/SearchFilter.styles.css

```diff
@@ -0,0 +1,210 @@
+.search-filter-container {
+  display: flex;
+  flex-direction: column;
+  width: 387px;
+  padding: 1rem;
+  height: 1002px;
+  border-radius: 1.2rem;
+}
+.search-filter-container .title {
+  width: 320px;
+  height: 53px;
+  margin-top: 1.5rem;
+}
+.search-filter-container h1 {
+  margin-top: 0.7rem;
+  margin-left: 1.5rem;
+  font-size: 20px;
+  line-height: 21px;
+  width: 340px;
+  height: 40px;
+  padding-top: 0.5rem;
+  padding-left: 2.5rem;
+  position: relative;
+  right: 2.1rem;
+  border-style: solid;
+  border-width: 0.1rem;
```

Comment:
Move to .search-filter-container .title

Reply:
I don't understand why I should move this to .search-filter-container .title

Reply:
Because then the border will be for the container that has the text and the arrow image